src/locales.json

```diff
@@ -368,6 +368,8 @@
     "en": {
       "troubleshooting": "Help to improve Integreat",
       "troubleshootingDescription": "Automatically send crash reports",
+      "allowPushNotifications": "Notifications",
+      "allowPushNotificationsDescription": "Allow notifications for the latest local updates and news",
```

Comment:
Maybe this should also be `Receive notifications...` instead of `Allow notificiations...`, same for german translation below. Not quite sure though.

Reply:
Yeah, these are from this discussion, not really a final conclusion yet: https://integreat.slack.com/archives/CDV9QF1GA/p1562231861030100